scripts/cert-manager/create-cluster.sh

```diff
@@ -0,0 +1,657 @@
+#!/bin/bash
```

Comment:
Create a new folder `tls` under /documentdb-playground and move all the scripts and docs under it.